src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -645,28 +645,30 @@ public FormValidation doCheckProjects(@AncestorInPath Job<?,?> project, @QueryPa
             boolean hasProjects = false;
             while(tokens.hasMoreTokens()) {
                 String projectName = tokens.nextToken().trim();
-                if (StringUtils.isNotBlank(projectName)) {
-                	Item item = Jenkins.getInstance().getItem(projectName,project,Item.class); // only works after version 1.410
-                    if(item==null){
-                        Item nearest = Items.findNearest(Job.class, projectName, Jenkins.getInstance());
-                        String alternative = nearest != null ? nearest.getRelativeNameFrom(project) : "?";
-                        return FormValidation.error(Messages.BuildTrigger_NoSuchProject(projectName, alternative));
-                    }
-                    if(!(item instanceof Job) || !(item instanceof ParameterizedJobMixIn.ParameterizedJob)) {
-                        return FormValidation.error(Messages.BuildTrigger_NotBuildable(projectName));
-                    }
+                if (StringUtils.isBlank(projectName)) {
+                    return FormValidation.error(Messages.BuildTrigger_NoSuchProject(" ", "?"));
```

Comment:
All other messages are using `hudson.tasks.Messages` from `core`. And most of them are localized. Is it ok to put custom message? It could break user's expectations.